aiven/client/cli.py

```diff
@@ -33,6 +33,8 @@
 import sys
 import time
 
+S = TypeVar("S", str, Optional[str])  # Must be exactly str or str | None
```

Comment:
isn't `str|None` enough? why do we need a union of `str` and `str|None`?

Reply:
`aiven-client` maintains compatibility with python 3.8